### PR TITLE
fixes borrow icons, reduces ajax calls for categories work_count

### PR DESCRIPTION
**Pull request by mekarpeles**

### CODE REVIEW

File: openlibrary/plugins/openlibrary/js/subjects.js [hunk 1]
```diff
@@ -98,36 +98,46 @@ $.extend(Subject.prototype, {
...
+        var format = work.public_scan ? 'public' : (work.printdisabled && !work.ia_collection.includes('inlibrary')) ? 'daisy' : 'borrow';
+        var bookread_url = work.public_scan ?
+            ('//archive.org/stream/' + work.ia + '?ref=ol') :
+            '/borrow/ia/' + work.ia;
+        var html = renderTag('div', {'class': 'coverMagic'},
+          renderTag('span', {
+              'itemtype': 'https://schema.org/Book',
+              'itemscope': ''},
+            renderTag('div', {'class': 'SRPCover'},
+              //renderTag('div', {'class': 'coverEbook'},
```
Comment (bfalling) [inline openlibrary/plugins/openlibrary/js/subjects.js]: Remove comment?

File: openlibrary/templates/home/categories.html [hunk 2]
```diff
@@ -46,46 +45,44 @@ <h2 class="home-h2">
...
-                Subject.count(cat, function(subject, count) {
-                    var cnt = count.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
-                    \$('[name="' + subject + '"]').text(cnt + ' books');
-                });
-            }
+
+            Subject.get_featured(function(results) {
+                for (var subject_key in results) {
+                    var subject = results[subject_key];
+                    console.log(subject);
```
Comment (bfalling) [inline openlibrary/templates/home/categories.html]: Use to keep? Or should remove?

File: openlibrary/templates/home/categories.html [hunk 3]
```diff
@@ -46,46 +45,44 @@ <h2 class="home-h2">
...
+                    var cnt = (subject.work_count || 0).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```
Comment (bfalling) [inline openlibrary/templates/home/categories.html]: Can you add a comment explaining the regexp?

Also, I suggest "count" over "cnt", an unnecessary and obfuscating abbreviation.

**Comment by bfalling** [comment]
Are these changes viewable on dev.openlibrary.org?

**Comment by mekarpeles** [comment]
Yes -- also, responded to these suggestions with 5b300b128141fcedff842d1d89c89925606bc373

**Comment by bfalling** [comment]
LGTM!

**Pull request merged**